custom_components/adjustable_bed/beds/leggett_okin.py

```
@@ -0,0 +1,358 @@
+            cancel_event=cancel_event,
+        )
+
+    async def start_notify(self, callback: Callable[[str, float], None]) -> None:
+        """Start listening for position notifications."""
+        self._notify_callback = callback
+        _LOGGER.debug("Leggett & Platt Okin beds don't support position notifications")
+
+    async def stop_notify(self) -> None:
+        """Stop listening for position notifications."""
+
+    async def read_positions(self, motor_count: int = 2) -> None:  # noqa: ARG002
```

Ruff (0.14.13)

Unused `noqa` directive (unused: `ARG002`)

Remove unused `noqa` directive

(RUF100)